war/package.json

```
@@ -15,7 +15,7 @@
     "build": "yarn prod",
     "start": "yarn dev --watch",
     "test": "jest --ci --config=jest.config.json --reporters=jest-standard-reporter --reporters=jest-junit",
-    "lint:js": "eslint . --ext js",
+    "lint:js": "eslint . --ext js -f checkstyle > target/eslint-warnings.xml",
```

I think using the shell redirection operator here is a bit fragile, as it assumes this command will be executed by a shell, which seems to be the case in the existing implementation but is not guaranteed to be the case. Better to use an explicit command-line argument which will work whether this command is executed by a shell or not:

```suggestion
    "lint:js": "eslint . --ext js -f checkstyle -o target/eslint-warnings.xml",
```